random/_types.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+/**
+ * Options for random number generation.
+ */
+export type RandomOptions = {
+  /**
+   * A function returning a random number between 0 and 1, used for
+   * randomization.
+   * @default {Math.random}
+   */
+  random: () => number;
```

Comment:
I think this option is not very web api-ish. I think it would be better to remove it and add some kind of `algorithm` option if needed,  that calls different functions internally. Doing `customRandomFunction() * (max - min) + min` seems very trivial.

Reply:
> I think this option is not very web api-ish. I think it would be better to remove it and add some kind of `algorithm` option if needed, that calls different functions internally. Doing `customRandomFunction() * (max - min) + min` seems very trivial.

Callbacks seem pretty web API-ish to me, and they're more versatile than specifying a name of an algorithm — with a callback, it's easy to use with third-party randomization functions/sources. But perhaps more importantly, specifying an algorithm name would surely make any and all of the all implemented algorithms into non-tree-shakeable dependencies of any function taking such options.

Reply:
@kt3k do we have a guide on how to name option functions? I thought thy have a suffix of `Fn` -> `randomFn`. Not sure though.